src/analysis/reaching-definitions-transfer-function.h

```diff
@@ -3,6 +3,7 @@
 
 #include "ir/find_all.h"
 #include "ir/local-graph.h"
+#include "lattice.h"
```

Comment:
I see we use `FinitePowersetLattice` below. Shouldn't we include `src/analysis/lattices/powerset.h`?

Reply:
Done